### ISSUE TITLE
Connectors configuration error after clean install

### ISSUE THREAD

**Issue by nicodonte** [reef-pi/reef-pi#2245]
![imagen](https://github.com/user-attachments/assets/61537ad8-6f61-45b8-b28b-986cc6f1f78d)
![imagen](https://github.com/user-attachments/assets/4c7e9975-1c0c-4d57-b1c6-cdc8e1b365b6)

Hi, I'm a new user on reef-pi, after installing it for the first time, when I try to enter the configuration menu for connectors I get an error and I can't add anything.

TypeError: this.props.driver.pinmap is null

    in i
    in div
    in div
    in div
    in div
    in div
    in i
    in Connect(i)
    in div
    in div
    in i
    in div
    in i
    in i
    in div
    in div
    in div
    in div
    in i
    in Connect(i)
    in i
    in a
    
What am I doing wrong or what am I missing? Could someone help me with this error?

Thanks.

**Comment by EmmettPeck** [comment]
Were you ever able to resolve this issue? I am encountering it as well on a fresh install following [reef-pi Guide 1](https://learn.adafruit.com/reef-pi-installation-and-configuration/raspberry-pi-configuration)

**Comment by nicodonte** [comment]
No. Unfortunately I couldn't solve it. Hopefully someone can help us.

**Comment by EmmettPeck** [comment]
It looks like Raspberry Pi Bookworm is not supported yet (#2193). Freshly downgrading to Bullseye via the RasPi Imager & following the guide fixed my issue! LED GO BLINK
